en/config/battery.md

```diff
@@ -0,0 +1,81 @@
+# Battery and Powermodule Setup
+
+> **Warning** Before you fly for the first time you should make sure you have a way to monitor your battery state. Not knowing how much fuel is left may likely lead to deep discharge of the used battery which can cause damage to the battery, loss of vehicle control and hence a crash.
+
+> **Note** The battery monitoring related features of PX4 can only be used if you have compatible hardware which is in most cases a power module that measures the battery voltage and the current between battery and vehicle.
+
+## Set the Basic Battery Settings
+
+1. Start *QGroundControl* and connect the vehicle.
+1. Select the **Gear** icon (Vehicle Setup) in the top toolbar and then **Power** in the sidebar.
+
+You are presented with the basic settings that characterize your battery.
+
+### Number of accumulator cells connected in series
+Batteries commonly rely on a chemical process to save and supply energy. The voltage across one atomic battery cell is given by the chemical properties of the battery type.
+For example the byy far most common battery type for remote controlled vehicles are lithium polymer (li-po) accumulators with a nominal cell voltage of 3.7V.
```

Comment:
typo 'byy'

What does nominal mean here - stated voltage on battery body? From what I read here the battery range for LiPo is about max 4.2 unloaded when full, and around 3.5V under load when "nearly empty". If the battery drops much lower (say 3.2V) then it is likely to be damaged. Correct?

Reply:
Yes that's correct, 3.7V is what is printed on every LiPo and what you generally assume the battery has when neglecting the variation because of the charge level and the load drop. It has the same meaning like 1.5V for your typical AA or AAA type alkaline batteries, they acutally don't have exactly this value but it's what you assume and use them for.